packages/react-examples/src/react-charting/AreaChart/AreaChart.Basic.Example.tsx

```diff
@@ -9,6 +9,8 @@ interface IAreaChartBasicState {
   height: number;
   isCalloutselected: boolean;
   showAxisTitles: boolean;
+  isHeightSliderFocused: boolean;
+  isWidthSliderFocused: boolean;
```

Comment:
Focus bar is not showing anywhere on the graph. Not only for width and height sliders